product_docs/docs/pgd/5/rel_notes/pgd_5.2.0_rel_notes.mdx

```diff
@@ -0,0 +1,72 @@
+| BDR       | 5.2.0   | Bug             | Add check for conflicting node names |
+| BDR       | 5.2.0   | Bug             | Fix a crash during Raft snapshot restore |
+| BDR       | 5.2.0   | Bug             | Ensure ACTIVE nodes do no longer attempt connecting to PARTING ones |
+| BDR       | 5.2.0   | Bug             | Fix `tcp_user_timeout` GUC to use the correct unit |
+| BDR       | 5.2.0   | Bug             | Correct v26 snapshots written by BDR 4 to be readable by 3.7 |
+| BDR       | 5.2.0   | Bug             | Modify bdr.subscription_stat for new columns |
+| BDR       | 5.2.0   | Bug             | miscellaneous fixes for PostgreSQL 16 compatibility |
+| BDR       | 5.2.0   | Bug             | Address a crash when BDR is used together with pgaudit |
+| BDR       | 5.2.0   | Bug             | Use the new async 2PC for group commit |
+| BDR       | 5.2.0   | Bug             | Skip parting synchronization to witness node |
+| BDR       | 5.2.0   | Bug             | Fix a bug in connection string construction |
+| BDR       | 5.2.0   | Bug             | Enable receiver side transaction tracking for non-streaming CAMO transactions |
+| BDR       | 5.2.0   | Bug             | Allow commit/abort prepared to create a transaction state |
+| BDR       | 5.2.0   | Bug             | Release tuple set for CAMO pre-commit |
+| BDR       | 5.2.0   | Bug             | Allow a logical join of node if there are foreign key constraints violations |
+| BDR       | 5.2.0   | Bug             | Multiple fixes for bdr.switch_node_group |
+| BDR       | 5.2.0   | Bug             | Only write consensus local state if it has changed |
+| BDR       | 5.2.0   | Bug             | Fixed deadlock on autopartition catalogs when a concurrent `DROP EXTENSION` is executing |
+| BDR       | 5.2.0   | Bug             | PA deadlocks when used with Group Commit |
+| BDR       | 5.2.0   | Bug             | Fix around Raft leader election in subgroup |
+| BDR       | 5.2.0   | Bug             | Fix sporadic failure when dropping extension after node restart |
+| BDR       | 5.2.0   | Bug             | pglogical receiver Segmentation fault |
+| BDR       | 5.2.0   | Bug             | Connection timeout and keepalive settings have wrong assumptions about the units (ms vs s) |
+| BDR       | 5.2.0   | Bug             | Fix deadlock between consensus and `run_on_all_nodes` queries |
+| BDR       | 5.2.0   | Bug             | Fix query cancellation propagatation across `bdr.run_on_all_nodes` |
+| BDR       | 5.2.0   | Bug             | Test support for HTTPS probe for HARP and PGD Proxy through TPAexec |
+| BDR       | 5.2.0   | Bug             | Update test views\_stats.sql for PA deadlock mitigation stats |
+| BDR       | 5.2.0   | Bug             | Group Commit vs Parallel Apply |
+| BDR       | 5.2.0   | Bug             | Fix appropriate tests to remove "bdr.writers\_per\_subscription =1" restriction when issue is resolved  |
```

Comment:
Please remove; bdr.writers_per_subscription=1 was temporary change for regression tests until some PA issues were addressed. So it is not a customer issue.